final_modeling.R

```diff
@@ -826,6 +894,296 @@ rsd_boxplot
 
 
 
+##################### REBUTTAL CODE - Fig 6 Multimouse home cage experiments ##################### 
+
+# unfiltered path
+# vid_path = "data/rebuttal/weight/"
+
+#filtered path
+vid_path = "data/rebuttal/weights_morphfiltered_and_expanded/"
```

Comment:
Might be worth describing where this data comes from (if it's not in the README)